pandas/tests/arithmetic/test_numeric.py

```diff
@@ -1054,7 +1054,7 @@ def test_ufunc_compat(self, holder):
         tm.assert_equal(result, expected)
 
     @pytest.mark.parametrize(
-        "holder", [pd.Int64Index, pd.UInt64Index, pd.Float64Index, Series]
+        "holder", [Int64Index, pd.UInt64Index, pd.Float64Index, Series]
```

Comment:
lets add these other Index classes at the top while we're here

Reply:
Thanks for the suggestion - I've added these too.